corehq/apps/repeaters/tests/test_repeater.py

```diff
@@ -237,6 +237,20 @@ def test_process_repeat_record_locking(self):
             check_repeaters()
             self.assertEqual(mock_process.delay.call_count, 2)
 
+    @run_with_all_backends
+    def test_cancel_repeat_record(self):
```

Comment:
nice test! can you rename it to something that suggests that this is testing the automatic cancelling of the record?